### PR TITLE
Troubleshooting section in docs

### PR DESCRIPTION
**Pull request by ayusharma**
Addition of troubleshooting section in cliquet docs.

### CODE REVIEW

File: cliquet_docs/troubleshooting.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+If you do not find a solution to your problem here, please
+:ref:`ask for help <communication_channels>`!
+
+
+ConnectionError: localhost:6379. nodename nor servname provided, or not known
+=============================================================================
+
+or
+
+IOError: [Errno 24] Too many open files
```
Comment (Natim) [inline cliquet_docs/troubleshooting.rst]: I would put both titles in the same paragraph title.

**Comment by Natim** [comment]
Thank you for that. I am wondering if we should add a way to display these errors in Kinto troubleshooting since they can appear to our Kinto users too.

Maybe the Kinto troubleshooting part can include this troubleshooting file and add its own trouble after the inclusion?

@leplatrem @almet thoughts?

File: cliquet_docs/troubleshooting.rst (hunk 1 above)
Comment (ayusharma) [inline cliquet_docs/troubleshooting.rst]: Okay, I'll make title in same paragraph.

**Comment by leplatrem** [comment]
The duplication between Cliquet docs and the Kinto product docs is a major concern indeed.

I started something using templates, we can probably think of something for glossary, trouble docs and other things.

Anyway, thanks a lot @ayusharma for this contribution!!

**Pull request merged**